packages/jest-phabricator/src/index.ts

```diff
@@ -3,25 +3,22 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- * @flow
  */
 
-import type {
-  AggregatedResult,
-  AggregatedResultWithoutCoverage,
-  CoverageMap,
-} from 'types/TestResult';
+import {TestResult} from '@jest/types';
 
-type PhabricatorReport = AggregatedResultWithoutCoverage & {
-  coverageMap?: ?CoverageMap,
+type PhabricatorReport = TestResult.AggregatedResultWithoutCoverage & {
+  coverageMap?: TestResult.CoverageMap | null;
 };
 
-function summarize(coverageMap: CoverageMap) {
+function summarize(coverageMap: TestResult.CoverageMap) {
   const summaries = Object.create(null);
 
-  coverageMap.files().forEach(file => {
+  coverageMap.files().forEach((file: string) => {
     const covered = [];
-    const lineCoverage = coverageMap.fileCoverageFor(file).getLineCoverage();
+    const lineCoverage: any = coverageMap
```

Comment:
> any

@SimenB Personally I dislike it. Maybe you have some ideas? 🤔

Reply:
pushed a fix by correcting the underlying types. They can probably be even stricter, but this works 🙂

Reply:
Thanks!